Tests/LibWeb/Layout/input/block-and-inline/float-max-content-containing-block-flex-display.html

```diff
@@ -0,0 +1,22 @@
+<style>
+    .display_flex {
+        display: flex;
+    }
+
+    div {
+        float: left;
+    }
+
+    span {
+        float: left;
+    }
+</style>
+
+<div class="display_flex">
+    <div>
+        <div>
+            <span>A</span>
+            <span>B</span>
+        </div>
+    </div>
+</div>
```

Comment:
```suggestion
</style><div class="display_flex"><div><div><span>A</span><span>B</span></div>
```

Reply:
Done.

Out of curiosity, why do you like it better as a one liner? Do you find the empty `TextNode`s annoying to read?

Reply:
yeah, I like it more when empty text nodes are not present in layout dump :)